### PR TITLE
Bumps for 3.24.0 RC1

### PR DESCRIPTION
**Pull request by AlvaroBrey**
I want to get the pdf scanner (#11201) into 3.24 as well, but I guess that should go to RC2 as it's still pending review.

- [x] Tests written, or not not needed

### CODE REVIEW

File: scripts/analysis/analysis-wrapper.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@ BUILD_NUMBER=$4
 PR_NUMBER=$5
 
 
-stableBranch="master"
+stableBranch="stable-3.24"
```
Comment (tobiasKaminsky) [inline scripts/analysis/analysis-wrapper.sh:10]: This should not be changed on master branch, but on stable-3.24 branch.

File: scripts/analysis/getBranchName.sh [hunk 2]
```diff
@@ -5,5 +5,5 @@
 if [ -z $3 ] ; then
     echo "master";
 else
-    curl 2>/dev/null -u $1:$2 https://api.github.com/repos/nextcloud/android/pulls/$3 | grep \"ref\": | grep -v '"master"' | cut -d"\"" -f4
+    curl 2>/dev/null -u $1:$2 https://api.github.com/repos/nextcloud/android/pulls/$3 | grep \"ref\": | grep -v '"stable-3.24"' | cut -d"\"" -f4
```
Comment (tobiasKaminsky) [inline scripts/analysis/getBranchName.sh:8]: Same here

**Review by tobiasKaminsky — CHANGES_REQUESTED** [review]
Wrong target branch?

**Comment by AlvaroBrey** [comment]
> Wrong target branch?

Yep, fixed. Good catch!

**Review by tobiasKaminsky — APPROVED** [review]

**Review by AndyScherzinger — APPROVED** [review]

**Pull request merged**